docker-compose.yml

```diff
@@ -190,3 +190,10 @@ volumes:
   mediawiki-mysql-data:
   elasticsearch-data:
   query-service-data:
+
```

Comment:
I would delete these last commented lines to keep docker-compose.yml clean, since they apply only to our local configuration. I would copy them into your local docker-compose-dev.yml, which is then not commited.

Reply:
done